test/e2e/tests/test_replicationgroup.py

```diff
@@ -133,6 +165,52 @@ def test_rg_cmd_fromsnapshot(self, rg_cmd_fromsnapshot):
         (reference, _) = rg_cmd_fromsnapshot
         assert k8s.wait_on_condition(reference, "ACK.ResourceSynced", "True", wait_periods=30)
 
+    # test update behavior of controller; this test can be changed to include multiple chained updates
+    def test_rg_cmd_update(self, rg_cmd_update_input, rg_cmd_update):
+        (reference, _) = rg_cmd_update
+        assert k8s.wait_on_condition(reference, "ACK.ResourceSynced", "True", wait_periods=30)
```

Comment:
TODO (@RedbackThomson): Move constants like "ACK.ResourceSynced" into consts in the `acktest` package.